src/components/drawer/drawer.js

```diff
@@ -0,0 +1,27 @@
+import template from "./drawer.html";
+import {
+  MDCPersistentDrawer,
+  MDCPersistentDrawerFoundation,
+  util
```

Comment:
if you do not use `util` and `MDCPersistentDrawerFoundation ` in this file -- there is no need to import them